lib/msal-common/test/cache/MockCache.ts

```diff
@@ -51,84 +64,38 @@ export class MockCache {
 
     // create account entries in the cache
     createAccountEntries(): void {
-        const accountData = {
-            username: "John Doe",
-            localAccountId: "object1234",
-            realm: "microsoft",
-            environment: "login.microsoftonline.com",
-            homeAccountId: "uid.utid",
-            authorityType: "MSSTS",
-            clientInfo: "eyJ1aWQiOiJ1aWQiLCAidXRpZCI6InV0aWQifQ==",
-        };
-        const account = CacheManager.toObject(new AccountEntity(), accountData);
+        const account = buildAccountFromIdTokenClaims(ID_TOKEN_CLAIMS, [
```

Comment:
Added this helper function since there's a lot of repetitive account creation across the test suite. I've updated all the account definitions that were touched on this PR but there are more instances that can benefit from this incrementally.